schematic/manifest/generator.py

```diff
@@ -1364,19 +1364,16 @@ def map_annotation_names_to_display_names(
         return annotations.rename(columns=label_map)
 
     def get_manifest_with_annotations(
-        self, annotations: pd.DataFrame, sheet_url:bool=None, strict: Optional[bool]=None,
```

Comment:
I remove sheet_url parameter from the function here because it is no longer needed.